src/main/java/edu/kis/powp/jobs2d/drivers/adapter/Jobs2dDriverToDrawerAdapter.java

```diff
@@ -0,0 +1,38 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
+
+/**
+ * Driver adapter to drawer.
+ */
+public class Jobs2dDriverToDrawerAdapter extends DrawerFeature implements Job2dDriver {
+    private int startX = 0, startY = 0;
+
+    public Jobs2dDriverToDrawerAdapter() {
+        super();
+    }
+
+    @Override
+    public void setPosition(int x, int y) {
+        this.startX = x;
+        this.startY = y;
+    }
+
+    @Override
+    public void operateTo(int x, int y) {
+        ILine line = LineFactory.getBasicLine();
```

Comment:
Unfortunately, this is not a generic solution (e.g. new line type requires new class, unnecessary direct coupling).